src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,82 @@
+package duke;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+
+import duke.command.Command;
+import duke.task.Task;
+
+public class Duke {
+
+    protected Storage storage;
+    protected TaskList tasks;
+    protected Ui ui;
+
+    /**
+     * Class constructor for Duke Class specifying the filepath
+     */
+    public Duke() {
+        ui = new Ui();
+        storage = new Storage(Paths.get(System.getProperty("user.dir"), "data", "duke.txt"));
```

Comment:
I like the way you get the file path! I should probably do this for my project instead of hardcoding it.